pkg/metrics/metrics.go

```diff
@@ -0,0 +1,44 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	runtimemetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+// When adding metric names, see https://prometheus.io/docs/practices/naming/#metric-names
+const (
+	existingKMMOCRsQuery = "kmmo_number_crs_total"
```

Comment:
```suggestion
	existingModuleCRsQuery = "module_number_crs_total"
```

I think you are only listing the `Module` objects.

Reply:
Same comment here. Also it it better to keep a prefix of kmmo for all the metrix, will make it much easier to find and group them

Reply:
With that I agree. How about `kmmo_module_number_crs_total`?

Reply:
Aren't you removing the `kmmo` prefix here?

Reply:
module is a the second "m" in kmmo